include/picongpu/particles/collision/detail/cellDensity.hpp

```diff
@@ -50,7 +50,7 @@ namespace picongpu::particles::collision::detail
                     auto particle = parAccess[partIdx];
                     density += particle[weighting_];
                 }
-                densityArray[linearIdx] = density / CELL_VOLUME;
```

Comment:
I would like to avoid it because we can not provide in this object interfaces for all typical cases. If wee need it shorter we should think about renamings in vector or provide a function `volume(sim.pic.getCellSize)` within picongpu.